sdk/eventhub/event-hubs/samples-dev/sendBufferedEvents.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/**
+ @summary Demonstrates how to send events to an Event Hub using the `EventHubBufferedProducerClient`.
+ * This sample is different from the one in `sendEvent.ts` in that the client manages batching of events and sending
+ * after a given amount of time or after a given amount of events are in a batch instead of you managing the same explicitly.
+ *
+ * @azsdk-weight 60
+ */
+
+import { AmqpAnnotatedMessage } from "@azure/core-amqp";
+import {
+  EventData,
+  EventHubBufferedProducerClient,
+  OnSendEventsErrorContext,
+} from "@azure/event-hubs";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env["EVENTHUB_CONNECTION_STRING"] || "";
+
+async function handleError(ctx: OnSendEventsErrorContext): Promise<void> {
+  console.log(`The following error occurred:`);
+  console.log(JSON.stringify(ctx.error, undefined, "  "));
```

Comment:
I usually see this written as `JSON.stringify(<object>, null, <number>)` where the number of spaces is given rather than a string of spaces.

Reply:
Why writing 4 characters when one can do!